Library/Homebrew/dependencies_helpers.rb

```diff
@@ -60,5 +90,4 @@ def dependents(formulae_or_casks)
       end
     end
   end
-  module_function :dependents
```

Comment:
This appears to be only for use in a test, so I updated the test, which i believe is preferable to using `module_function`.